### PR TITLE
Adding image_pull_secret property to verifier

### PR DESCRIPTION
**Pull request by vcanaa**
This property can be used to tell the verifier what imagePullSecret to use.

This change is needed by on-prem verification.

### CODE REVIEW

File: scripts/verify [hunk 1]
```diff
@@ -92,6 +96,13 @@ echo "INFO Parameters: $parameters"
 echo "INFO Creates namespace \"$NAMESPACE\""
 kubectl create namespace "$NAMESPACE"
 
+if [[ -n "$image_pull_secret" ]]; then
+  echo "Copying secret/$image_pull_secret from namespace/$default to namespace/$NAMESPACE."
+  kubectl get secret "$image_pull_secret" --namespace=default --export -o=yaml | kubectl apply --namespace="$NAMESPACE" -f -
```
Comment (huyhg) [inline scripts/verify]: use `create` instead of `apply` to avoid clobbering existing resources

File: scripts/verify [hunk 2]
```diff
@@ -40,6 +40,10 @@ case $i in
     additional_deployer_role="${i#*=}"
     shift
     ;;
+  --image_pull_secret=*)
+    image_pull_secret="${i#*=}"
```
Comment (huyhg) [inline scripts/verify]: per offline discussion: image_pull_secret_name

File: scripts/verify [hunk 3]
```diff
@@ -146,6 +157,7 @@ echo "INFO Initializes the deployer container which will deploy all the applicat
   --deployer="$deployer" \
   --parameters="$parameters" \
   --entrypoint='/bin/deploy_with_tests.sh' \
+  --image_pull_secret="$image_pull_secret" \
```
Comment (huyhg) [inline scripts/verify]: we shouldn't need this if the namespace is already annotated?

File: scripts/verify (hunk 2 above)
Comment (vcanaa) [reply scripts/verify]: Per further discussion keeping image_pull_secret to keep consistent with the same parameter in provision.py

**Pull request merged**